grammar_test/expressions.txt

```diff
@@ -866,3 +866,15 @@ foo(...rest)
 (program
   (expression_statement
     (function_call (identifier) (arguments (rest_argument (identifier))))))
+
+==============================================
+Math expressions
+==============================================
+
+(foo - bar) / baz
```

Comment:
Did this fail w/ the old one? I thought that for various reasons, the whole thing actually had to be in parentheses for it to fail.